integrationExamples/gpt/audigentSegments_example.html

```diff
@@ -0,0 +1,262 @@
+                        storage: {
+                            type: "cookie",
+                            name: "pubcid",
+                            expires: 365
+                        },
+                        // value: {
+                        //     foo: '9879878907987',
+                        //     bar:'93939'
+                        // }
+                    },  {
+                      name: 'identityLink',
+                      params: {
+                        pid: '14' // Set your real identityLink placement ID here
+                      },
+                      storage: {
+                        type: 'cookie',
+                        name: 'idl_env',
+                        expires: 30
+                      }
+                    }],
+                    syncDelay: 5000,
+                    auctionDelay: 1000 
+                },
+                realTimeData: {
+                    auctionDelay: 1000,
+                    dataProviders: [{name: "audigent"}]
+                }
+            });
+            pbjs.addAdUnits(adUnits);
+            pbjs.requestBids({bidsBackHandler: sendAdserverRequest});
+        });
+
+        function sendAdserverRequest() {
+            document.getElementById('tdid').innerHTML = adUnits[0].bids[0].userId['tdid'];
+            document.getElementById('audigent_segments').innerHTML = JSON.stringify(adUnits[0].bids[0].realTimeData.audigent_segments);
+
+            if (pbjs.adserverRequestSent) return;
+            pbjs.adserverRequestSent = true;
+            googletag.cmd.push(function() {
+                pbjs.que.push(function() {
+                    pbjs.setTargetingForGPTAsync();
+                    googletag.pubads().refresh();
+                });
+            });
+        }
+
+        setTimeout(function() {
+            sendAdserverRequest();
+        }, FAILSAFE_TIMEOUT);
+    </script>
+
+    <script>
+        (function () {
+            var gads = document.createElement('script');
+            gads.async = true;
+            gads.type = 'text/javascript';
+            var useSSL = 'https:' == document.location.protocol;
+            gads.src = (useSSL ? 'https:' : 'http:') +
+                '//www.googletagservices.com/tag/js/gpt.js';
+            var node = document.getElementsByTagName('script')[0];
+            node.parentNode.insertBefore(gads, node);
+        })();
+    </script>
+
+    <script>
+        googletag.cmd.push(function() {
+            googletag.defineSlot('/112115922/FL_PB_MedRect', [[300, 250],[300,600]], 'test-div').addService(googletag.pubads());
+            googletag.pubads().enableSingleRequest();
+            googletag.enableServices();
+        });
+    </script>
+</head>
+
+<body>
+<h2>Audigent Segments Prebid</h2>
+
+<div id='test-div'>
+    <script>
+        googletag.cmd.push(function() { googletag.display('test-div'); });
+    </script>
+</div>
+TDID:
+<div id='tdid'>
+</div>
+
+Audigent Segments:
+<div id='audigent_segments'>
```

Comment:
Don't understand why a div is needed.

Reply:
This is just here to show the resulting segments that could be attached to the bid request objects.